models/articles.models.js

```diff
@@ -31,6 +32,39 @@ function selectArticleById(article_id) {
     });
 }
 
+async function updateArticleById(article_id, inc_votes) {
+  if (inc_votes === undefined) {
+    return Promise.reject({ status: 400, msg: "Malformed Request" });
+  }
+  if (isNaN(article_id)) {
+    return Promise.reject({ status: 400, msg: "Id NaN" });
+  }
+  //   check if article_id exist in database
+  async function checkIfArticle_idExist(article_id) {
```

Comment:
This is handled by psql so could be refactored to be caught instead in your psql error handler